### PR TITLE
Get rid of spurious cmp in definition.

### PR DESCRIPTION
**Pull request by angerhang**
See #52 .

### CODE REVIEW

File: lib/LaTeXML/Package/statements.sty.ltxml [hunk 1]
```diff
@@ -159,7 +159,7 @@ sub definitionBody {
       $doc->openElement('dc:title');
       $doc->absorb($title);
       $doc->closeElement('dc:title');}
-    $doc->openElement('omdoc:CMP');
+   if ($doc->isCloseable('omdoc:CMP')) { $doc->openElement('omdoc:CMP'); }
```
Comment (kohlhase) [inline lib/LaTeXML/Package/statements.sty.ltxml]: I do not understand that do we really want to open the element if it is closable?

File: lib/LaTeXML/Package/statements.sty.ltxml [hunk 2]
```diff
@@ -159,7 +159,7 @@ sub definitionBody {
...
    $doc->absorb($props{body}) if $props{body};
    $doc->maybeCloseElement('omdoc:CMP');
```
Comment (kohlhase) [inline lib/LaTeXML/Package/statements.sty.ltxml]: does the ```maybeCloseElement``` now exist after all? I thought Bruce did not want to introduce it.

Comment (angerhang) [inline lib/LaTeXML/Package/statements.sty.ltxml]: Yes maybeCloseElement does exist. See [LaTeXML::Core:Document](http://dlmf.nist.gov/LaTeXML/manual/coremodules/LaTeXML_Core_Document.html). 

I think it is `maybeOpenElement` that he doesn't want to introduce.

File: lib/LaTeXML/Package/statements.sty.ltxml (hunk 1 above)
Comment (angerhang) [inline lib/LaTeXML/Package/statements.sty.ltxml]: In fact it was mysterious to me too. I don't know why this change also yields the right result. It might have some side effects that I didn't notice. But https://github.com/KWARC/LaTeXML-Plugin-sTeX/pull/59 makes more sense now.
